dpd-client/build.rs

```diff
@@ -84,7 +84,10 @@ fn main() -> Result<()> {
                 |state: &crate::ClientState, result: &Result<_, _>| {
                     slog::debug!(state.log, "client response"; "result" => ?result);
                 }
-            }),
+            })
+            .with_replacement("Ipv4Cidr", "crate::Ipv4Cidr", std::iter::empty())
+            .with_replacement("Ipv6Cidr", "crate::Ipv6Cidr", std::iter::empty())
+            .with_replacement("Cidr", "crate::Cidr", std::iter::empty()),
```

Comment:
Is this replacing with the corresponding Nexus types? I think we might want to avoid that, if so. When we can actually use the existing `dpd-client` from Dendrite, it'd be handy to be able to swap it in, without changing much else. Using these replacement types would mean we'd need to change all those call sites.

Fine to defer, I think, but maybe add an issue so we don't forget that, or a note in the existing issue for moving to the existing Dendrite client.